Gemfile

```diff
@@ -2,6 +2,7 @@ source ENV['GEM_SOURCE'] || "https://rubygems.org"
 
 gem "hiera", :path => File.dirname(__FILE__), :require => false
 gem 'deep_merge', :require => false
+gem 'hocon', "~> 1.1.0"
```

Comment:
If this is a hard dependency, we'll need to add it to `.gemspec` and `ext/project_data.yaml` as well.